backend/server/tests/unit/test_schema.py

```diff
@@ -161,15 +169,19 @@ def test_fetch_yearly_predictions(self):
 
     def test_fetch_latest_round_predictions(self):
         ml_models = list(MLModel.objects.all())
-        year = date.today().year
+        year = 2017
```

Comment:
when I used hardcoded values in test I prefer to use YEAR, uppercase. So then I see easily when/where I'm using hardcoded values.